antiweb.egg-info/SOURCES.txt

```diff
@@ -8,37 +8,4 @@ antiweb.egg-info/dependency_links.txt
 antiweb.egg-info/requires.txt
 antiweb.egg-info/top_level.txt
 antiweb.egg-info/zip-safe
-tests/block1.c
```

Comment:
antiweb.egg-info/SOURCES.txt is only used for metadata when building a zipped distribution. The recent built distribution deleted those files automatically.